docs/content/guides/columns/column-filter.md

```diff
@@ -155,765 +450,2860 @@ import 'handsontable/dist/handsontable.full.min.css';
+          sellTime: '01:23 AM',
+          inStock: false,
+        },
+        {
+          brand: 'Gigabox',
+          model: 'HL Mountain Frame',
+          price: 1890.9,
+          sellDate: 'May 3, 2023',
+          sellTime: '11:27 AM',
+          inStock: false,
+        },
+        {
+          brand: 'Camido',
+          model: 'Cycling Cap',
+          price: 130.1,
+          sellDate: 'Mar 27, 2023',
+          sellTime: '03:17 AM',
+          inStock: true,
+        },
+        {
+          brand: 'Chatterpoint',
+          model: 'Road Tire Tube',
+          price: 59,
+          sellDate: 'Aug 28, 2023',
+          sellTime: '08:01 AM',
+          inStock: true,
+        },
+        {
+          brand: 'Eidel',
+          model: 'HL Road Tire',
+          price: 279.99,
+          sellDate: 'Oct 2, 2023',
+          sellTime: '01:23 AM',
+          inStock: true,
+        },
+      ]}
+      columns={[
+        {
+          title: 'Brand',
+          type: 'text',
+          data: 'brand',
+        },
+        {
+          title: 'Model',
+          type: 'text',
+          data: 'model',
+        },
+        {
+          title: 'Price',
+          type: 'numeric',
+          data: 'price',
+          numericFormat: {
+            pattern: '$ 0,0.00',
+            culture: 'en-US',
+          },
+          className: 'htLeft',
+        },
+        {
+          title: 'Date',
+          type: 'date',
+          data: 'sellDate',
+          dateFormat: 'MMM D, YYYY',
+          correctFormat: true,
+          className: 'htRight',
+        },
+        {
+          title: 'Time',
+          type: 'time',
+          data: 'sellTime',
+          timeFormat: 'hh:mm A',
+          correctFormat: true,
+          className: 'htRight',
+        },
+        {
+          title: 'In stock',
+          type: 'checkbox',
+          data: 'inStock',
+          className: 'htCenter',
+        },
+      ]}
+      // enable the column menu
+      dropdownMenu={true}
+      // enable filtering
+      filters={true}
+      // to differentiate this example's CSS from other examples on this page
+      className="custom-filter-button-example-2"
+      height="auto"
+      licenseKey="non-commercial-and-evaluation"
+    />
+  );
+};
 
-  const selectEntry = (rowEntry) => {
-    if (selectedRowEntries.includes(rowEntry)) {
-      setSelectedRowEntries(selectedRowEntries.filter(entry => entry !== rowEntry));
+/* start:skip-in-preview */
+ReactDOM.render(<App />, document.getElementById('exampleCustomFilterButton2'));
+/* end:skip-in-preview */
+```
 
-    } else {
-      setSelectedRowEntries([...selectedRowEntries, rowEntry]);
-    }
+```css
+/* hide the column menu button by default */
+.custom-filter-button-example-2 .changeType {
+  visibility: hidden;
+}
+
+/* show the column menu button on hover */
+.custom-filter-button-example-2 .relative:hover .changeType {
+  visibility: visible;
+}
+```
+
+:::
+
+:::
+
+## Change the width of the filter menu
```

Comment:
A demo would be nice here. It's an issue when the long text is cropped in the filter values preview.

Reply:
We came across a problem here: an older method of changing the menu width (through `modifyColWidth`) doesn't work anymore. And changing the menu width through CSS doesn't work with the `className` option, so the styling changes would affect all demos on the page. @adrianszymanski89 can you confirm?